testctrl/svc/store/storage_server.go

```diff
@@ -75,16 +89,19 @@ func (s *StorageServer) StoreEvent(sessionName string, event *types.Event) error
 	return fmt.Errorf("Unknown session name: %s", sessionName)
 }
 
-// GetLatestEvent returns the latest event associated with an existing session.
-func (s *StorageServer) GetLatestEvent(sessionName string) *types.Event {
+// GetLatestEvent returns the latest event associated with an existing session,
+// and an error if the session does not exist.
+func (s *StorageServer) GetLatestEvent(sessionName string) (*types.Event, error) {
 	s.mutex.Lock()
 	defer s.mutex.Unlock()
 	eventStore, sessionExists := s.eventMap[sessionName]
-	if !(sessionExists && len(eventStore) > 0) {
-		return nil
+	if !sessionExists {
+		return nil, fmt.Errorf("Unknown session name: %s", sessionName)
```

Comment:
nit: errors are usually lowercase for chaining

```suggestion
		return nil, fmt.Errorf("unknown session name: %s", sessionName)
```

Reply:
Done.